### ISSUE TITLE
Add back "Copy Selected Row with Current MD5" to Proper Noun replacements

### ISSUE THREAD

**Issue by miepo** [HIllya51/LunaTranslator#984]
### Expected behavior 预期的功能

![Clipboard_08-11-2024_01](https://github.com/user-attachments/assets/a8f680e7-cd65-4311-b5f0-aaf96649c64a)

1. This setting was removed after v5.16.2. It was very useful to quickly make copies of existing Proper Noun replacements with a different MD5. It would be nice if this feature could be added back since it was very useful for entry duplication.
There is no easy way to make shared replacements when 2+ VNs have the same characters like in sequels and fandisks. If I create replacements for VN1 and want to use them in VN2, VN3 etc. it is not possible to easily copy all replacements and only change the MD5s. "Copy Selected Row with Current MD5" was very useful to make copies with the correct syntax that I would then manually edit in noundictconfig.json with notepad++.

2. At first I wanted to ask for two buttons to move an entry to the top and bottom. But even better would be a new "Move to..." button. On click it asks for the position to move the entry to. What do you think?

**Comment by HIllya51** [comment]
![屏幕截图 2024-08-12 031732](https://github.com/user-attachments/assets/d561477d-be1a-457f-850c-cd9ecc682bfc)

![屏幕截图 2024-08-12 031743](https://github.com/user-attachments/assets/82f82759-8471-460c-89ef-57caf506a15c)
![屏幕截图 2024-08-12 031758](https://github.com/user-attachments/assets/297d5a44-e33e-4eed-96ee-c50deea78954)

**Comment by miepo** [comment]
@HIllya51 I think I understood your screenshots. I can copy-paste entries from Proper Noun replacements to "Translation Optimization" of a different VN. Very useful in the future. Two more questions about this:

1. What file/location does Luna Translator save a game's "Translation Optimizations" to? It doesn't seem to be in the userconfig folder.

2. I understand how the "Follow Default" toggle in Game Settings works, but what does "Inherit Default" do exactly? I tested this: I made a global Proper Noun replacement (杏 as Anzu) and a replacement in the game setting (杏 as AAA). 

a. "Follow Default" on = translation shows 杏 as Anzu; Proper Noun replacement takes precedence
b. "Follow Default" off + "Inherit Default" off = translation shows 杏 as AAA; game setting takes precedence
c. "Follow Default" off + "Inherit Default" on = translation shows 杏 as Anne; seems to ignore any replacements and uses DeepL's unmodified translation

Is "Inherit Default" broken or what does it do? Is it meant to disable all replacements for a game?
